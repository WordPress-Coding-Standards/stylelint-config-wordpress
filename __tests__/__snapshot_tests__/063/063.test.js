'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '063', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '063' )],
		configFile: systemTestUtils.caseConfig( '063' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
