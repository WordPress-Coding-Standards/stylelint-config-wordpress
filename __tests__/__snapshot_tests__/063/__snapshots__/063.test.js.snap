// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`063 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "063/stylesheet.scss",
    "warnings": Array [
      Object {
        "column": 4,
        "line": 110,
        "rule": "at-rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before at-rule (at-rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 34,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 1,
        "line": 47,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 1,
        "line": 55,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 1,
        "line": 63,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 1,
        "line": 71,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 1,
        "line": 79,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 1,
        "line": 88,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 1,
        "line": 95,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 1,
        "line": 105,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 2,
        "line": 107,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@if\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 4,
        "line": 110,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@else\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 1,
        "line": 120,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 1,
        "line": 145,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@mixin\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 6,
        "line": 331,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 529,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 536,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 8,
        "line": 663,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 6,
        "line": 808,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 6,
        "line": 960,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 3,
        "line": 1005,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 3,
        "line": 1006,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 4,
        "line": 1015,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 6,
        "line": 1112,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 7,
        "line": 1180,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 4,
        "line": 1238,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 1379,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 6,
        "line": 1538,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 1545,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 1585,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 1618,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 6,
        "line": 1633,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 6,
        "line": 1650,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 4,
        "line": 1689,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 1753,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 1754,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 4,
        "line": 1838,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 1957,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 4,
        "line": 2049,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 2057,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 2065,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 5,
        "line": 2084,
        "rule": "at-rule-no-unknown",
        "severity": "error",
        "text": "Unexpected unknown at-rule \\"@include\\" (at-rule-no-unknown)",
      },
      Object {
        "column": 3,
        "line": 110,
        "rule": "block-closing-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"}\\" (block-closing-brace-newline-after)",
      },
    ],
  },
]
`;
